e2etest/declarativeScenario.go

```diff
@@ -90,7 +91,8 @@ func (s *scenario) Run() {
 	// resume if needed
 	if s.needResume {
 		// TODO: create a method something like runAzCopy, but which does a resume, based on the job id returned inside runAzCopy
-		s.a.Error("Resume support is not built yet")
+		// s.a.Error("Resume support is not built yet")
```

Comment:
Should the error message still stay if there is no implementation yet?

Reply:
I was actually working on the tests suite between two PRs and sharing the changes. This is fine to leave, once the folder resume PR gets merged.